Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -188,18 +188,26 @@ open class Player: BaseObject {
 
     private func bindPlaybackEvents() {
         if let playback = core?.activePlayback {
+            var listenId = ""
```

Comment:
I think we don't need to share this variable. We can keep it inside for context. What do you think?

Reply:
We share so we don't have to define it every iteration and then again outside the for loop.